samples/project-anfield/export_cluster_config.py

```diff
@@ -129,7 +128,9 @@ def get_whitelist_settings(cohesity_client):
     "sql_entity_mapping": library.get_sql_entity_mapping(cohesity_client,
                                                          env_enum.KSQL),
     "ad_entity_mapping": library.get_ad_entity_mapping(cohesity_client,
-                                                         env_enum.KAD)
+                                                         env_enum.KAD),
+    "vlans": library.get_vlans(cohesity_client),
```

Comment:
something doesn't add up: 
1. while exporting you export all these by default 
2. While importing you import selectively. 

What if there is sensitive information and customer doesn't want to export it by default ?